### ISSUE TITLE
Opinion: Why i LOVE Apple and could LOVE HelloSystem

### ISSUE THREAD

**Issue by jhjacobs81** [helloSystem/hello#168]
Let me first say that i have a Linux background. I tried System 7, didnt like it. Moved to windows, loathed it. Moved to Linux. In the mean time, my opinion was that Apple was too expensive, too closed, too much of all the bad thoughts i had about it. Never would i use Apple.

I LOOOHOOVED to thinker around, spending many nights figuring out how to make things work, and i LOOHOOVE the idea of OSS. I got  a DELL, an Android, and some cheap tablet, and for many years i was a happy camper. I worked in IT, so all the tinkering made me a quick fellow to work with, my Google-Fu improved by the minute. It was all blissful existance for me. 

Then my Android got stolen, and as a replacement i was send off with an iPhone 3. The horrors of my worst nightmare’s had come true! But allas. I was a grown man, i would withstand the storms. I would be strong.

OH! MY! GOD! Did i fall in love with Apple quickly. The phone indeed “just works” and it was so very intuitive. Everything was easy to navigate, and i quickly realized i never wanted back to Android again. I quickly switched my tabled out for an iPad, then came the mac mini, the imac, the macbooks.. the Apple TV and ofcourse the Apple watch. The integration, the senergy, the ease of making all these devices work together, AHHH i was free from the eternal tinkering, and i quickly discovered that i didnt like to tinker as much as i thought i did. I like to be creative. But i’m also a perfectionist. I cant stand things that work “half” or look “messy”. I like uniformity. I like perfection. I could be very well the first BORG to ever exist. If only i could convince my family to link their brains with me.. 

And yet.. i still liked the idea of OSS. And while Apple is acting like a benevolent dictator, its still a dictator. Working in IT Security, privacy is a huge thing for me. And while these are actually the same reasons why i “convert” those around me to Apple, if Apple ever was to go OSS with its *os platforms, i would quickly swap out their cloud service for my own. I like how all my devices look, and i like how easily they sync and work. I do not like that i am depending on some big American company that, despite its best intentions, is still an american company, with its shareholders and other money hungry people that would sell my data at the first oportunity they ever could. But truthfully, there is nothing out there that matches Apple’s drive for perfection, save for my own. I tried working with other OSS products and projects, but i quickly fall tired with those because they are usually a far cry from what Apple represents to me. I tried some Darwin clone but the project seems dead, and im only just now starting to learn Golang, so im far, far away from launching my own clone :D 

simply put, use Apple producs because they are the prettiest of all evils. 

And now i came across this github respository.. oh my im one step away from orgasming! I think, for the first time, i found a project i actually want to help :) if HelloSystem could free me from dictatorship, while at the same time strive for that same perfectionism.. then i want to be part of it! “Just works” is half of it for me, “looks perfect” is the other half :) 

unfortunatly, i have no experience in programming, other then my helloworld commandline program i learned following a golang tutorial! But i have a keen eye for design.  I have plenty of idea’s!

So, how can i help? Is there a discord server for this project, or an IRC channel or anything i could join?

**Comment by linuxkettle** [comment]
> So, how can i help? Is there a discord server for this project, or an IRC channel or anything i could join?

From what I can see, everything is done through GitHub issues, although both of those would be great ideas. Also, @probonopd  could start a [GitHub Discussion](https://docs.github.com/en/discussions) for design talk and enhancement ideas, instead of cluttering the issue sections of the likes of Filer, hello etc.

**Comment by jhjacobs81** [comment]
Or maybe build a website for it :) i was suprised to see hellosystem.io is still available to register! I was tempted to go ahead and register it, but i would not want to intrude :)

**Comment by jhjacobs81** [comment]
I also firmly believe someone should write a manifesto, its one thing Debian did right. And it lays the foundation for the future. For other dev’s to understand why and how helloSystem came to be, and it keeps the project from deviating from its core value’s as more and more people find their way to this project :) and it helps with securing funding..

**Comment by probonopd** [comment]
> From what I can see, everything is done through GitHub issues, although both of those would be great ideas. Also, @probonopd could start a GitHub Discussion for design talk and enhancement ideas, instead of cluttering the issue sections of the likes of Filer, hello etc.

We are using this project for general/"philosophical" topics/ideas not directly related to existing code:
https://github.com/helloSystem/hello. So you've come to the right place. Once GitHub allows GitHub Discussions for an entire organization (helloSystem), we will use that.

We are using the other projects like Filer,... for concrete issues related to the code.

> Is there a discord server for this project, or an IRC channel or anything i could join?

Please see https://hellosystem.github.io/docs/developer/contact

> Or maybe build a website for it :)

https://hellosystem.github.io/docs
